pkg/webhook/connection/connection_webhook.go

```diff
@@ -142,6 +142,37 @@ func (w *Webhook) ValidateDelete(ctx context.Context, obj runtime.Object) (admis
 	if len(extAttachments.Items) > 0 {
 		return nil, errors.Errorf("connection has external attachments")
 	}
+	if conn.Spec.Management != nil {
+		mgmtConnList := &wiringapi.ConnectionList{}
+		if err := w.Client.List(ctx, mgmtConnList, client.MatchingLabels{
+			wiringapi.LabelConnectionType: wiringapi.ConnectionTypeManagement,
+		}); err != nil {
+			return nil, errors.Errorf("error listing servers on management connection")
```

Comment:
wrong error message - it's listing mgmt connections here

Reply:
will do